<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiverDialog</class>
 <widget class="QWidget" name="ReceiverDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="serviceOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>771</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="serviceOutputLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Output:</string>
   </property>
  </widget>
  <widget class="QLabel" name="serviceListLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>258</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Subscribed services:</string>
   </property>
  </widget>
  <widget class="QLabel" name="senderTableLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sender:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="senderTable">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>31</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name of Process</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Port</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PID</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="serviceList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>256</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="receiverAllServices">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>See output of all subscribed services</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopListeningButton">
   <property name="geometry">
    <rect>
     <x>671</x>
     <y>401</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop listening</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startListeningButton">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>401</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start listening</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
